pkgs/by-name/no/notion-app/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  unzip,
+  testers,
+  libplist,
+  jq,
+  writeText,
+  runCommand,
+}:
+let
+  info =
+    (builtins.fromJSON (builtins.readFile ./info.json))."${stdenvNoCC.targetPlatform.system}"
+      or (throw "notion-app: unsupported system ${stdenvNoCC.targetPlatform.system}");
+in
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "notion-app";
+  version = info.version;
+  src = fetchurl { inherit (info) url hash; };
+  sourceRoot = ".";
```

Comment:
Redundant
```suggestion
```

Reply:
This is needed, if I don't add this, I need add `cd ..` after unpackPhase.

Reply:
Well, I mean, the source is a ZIP file anyway so you should use `fetchzip` instead of relying on the default unpack phase

Reply:
I can't use fetchzip because it will throw a error
```
error: hash mismatch in fixed-output derivation
```

It will check hash for content rather than zip file

Reply:
Well you can just update the hash to what `fetchzip` outputs 😓

Reply:
No. I can’t. Because this hash is from notion’s update info yaml file.

Reply:
Hm. Alright then, ignore me xD

Reply:
Hm. Alright then, ignore me xD